starfish/pipeline/filter/bandpass.py

```diff
@@ -85,9 +97,12 @@ def filter(self, stack: ImageStack, in_place: bool=True) -> Optional[ImageStack]
 
         """
         bandpass_ = partial(
```

Comment:
if you're going in this direction, i would rather see:

```
        bandpass_ = partial(
            self.bandpass,
            lshort=self.lshort, llong=self.llong, threshold=self.threshold, truncate=self.truncate
        )
```

first argument to `partial` is more different than the rest of the arguments than truncate is to the first arguments.